components/automate-chef-io/content/docs/iam-v2-guide.md

```diff
@@ -49,21 +29,14 @@ This policy display includes the following:
 
 ## Policy Conversion
 
-After upgrading, we recommend that you reconstitute your v1 policies as v2 policies.
+We recommend that you reconstitute your v1 policies as v2 policies.
```

Comment:
okay so its a bit tricky i think. because many folks have tokens that need to be moved to the admin or ingest policy and if they just delete all their legacy policies those tokens will stop working. ill try and give this a pass to address some of the issues you raised though.